docs.md

```diff
@@ -25,14 +24,16 @@ layout: page
     <ul>
         <li><a href="https://cli.dhis2.nu">d2</a> - command-line interface for DHIS2 development</li>
         <li><a href="https://cli-style.dhis2.nu">d2-style</a> - standardized code and commit style enforcement</li>
-        <!-- <li><a href="https://cli-utils-cypress.dhis2.nu">d2-utils-cypress</a> - boilerplate wrapper for Cypress integration and end-to-end testing</li> -->
         <li><a href="https://cli-utils-docsite.dhis2.nu">d2-utils-docsite</a> - generate a documentation website with consistent structure and design</li>
     </ul>
     <h3>Legacy libraries</h3>
-    <p>These libraries still receive maintenance support, but will be deprecated at the end of 2020.  They should be used sparingly in modern DHIS2 applications.</p>
+    <p>These libraries still receive maintenance support, but will be deprecated at the end of 2020. They should be used sparingly in modern DHIS2 applications.</p>
```

Comment:
I think this is close but wrong. I'm not aware we'll deprecate `core` and/or `widgets`. There's a (high?) chance that we might split `widgets` into several packages, but does that automatically mean we'll drop support for `core` as well?

Reply:
Actually yeah, this is wrong. ui-core, ui-widgets and ui-forms aren't deprecated at all, just the old repos. I'll remove those changes.

Reply:
Fixed in 4ca740ba2c3368be9e228833f9431d5b58e167ac